contracts/minting/MintController.sol

```diff
@@ -100,6 +101,23 @@ contract MintController is Controller {
         return internal_setMinterAllowance(minter, newAllowance);
     }
 
+    /**
+     * @dev decreases the minter allowance if and only if the minter is
+     * currently active. The controller can safely send a signed decrementMinterAllowance()
+     * transaction to a minter and not worry about it being used to undo a removeMinter()
+     * transaction.
+     */
+    function decrementMinterAllowance(uint256 allowanceDecrement) onlyController public returns (bool) {
+        address minter = controllers[msg.sender];
+        require(minterManager.isMinter(minter));
+
+        uint256 currentAllowance = minterManager.minterAllowance(minter);
+        uint256 newAllowance = currentAllowance.sub(allowanceDecrement);
+
```

Comment:
If the currentAllowance is less than the allowanceDecrement, the contract should set the allowance to 0.  Issuers are going to have pre-signed incrementMinterAllowance and decrementMinterAllowance transactions.  They need a way to drive the allowance down to 0 regardless of the current allowance value.  The sub() function throws if allowanceDecrement>currentAllowance.

We can use the OpenZeppelin Math library:
`uint256 newAllowance = max(currentAllowance, allowanceDecrement).sub(allowanceDecrement)`

Reply:
Yeah, you're right! that's fixed now

Reply:
done

Reply:
Yeah, you're right. The event is now firing the actual decremented amount